### DISCUSSION TITLE
New downloads all-time high!

### DISCUSSION THREAD

**Discussion by SkalskiP** [roboflow/supervision#239] [Announcements]
`supervision` broke daily downloads all-time high. `6.206` -> `7.826` :rocket:

![Screenshot 2023-07-25 at 09 25 54](https://github.com/roboflow/supervision/assets/26109316/d44548ad-537f-45b4-bd27-19f7c98822c0)

**Comment by hardikdava** [comment]
hi @SkalskiP 👋 , we are on right track. We are adding tools which community needs at a single place.

**Comment by SkalskiP** [comment]
I'm not sure what is going on. I'm not sure it is sustainable. 🤷🏻‍♂️ But we have a new all-time high, breaking the record second day in a row. `7.826` -> `11.859` 

<img width="728" alt="Screenshot 2023-07-26 at 10 11 20" src="https://github.com/roboflow/supervision/assets/26109316/83b6a438-b343-4eed-864b-ee51dd1af00c">

**Reply by hardikdava** [reply]
@SkalskiP here is a start count progress over the time. ![star-history-2023726](https://github.com/roboflow/supervision/assets/39372750/0a70b95d-dc24-4775-99e3-e1c87c5aae89)
